api/router.go

```diff
@@ -162,14 +162,24 @@ func NewRouter(s *Server) *mux.Router {
 	r.Methods("POST").Path("/v1/backingimages").Handler(f(schemas, s.BackingImageCreate))
 	r.Methods("DELETE").Path("/v1/backingimages/{name}").Handler(f(schemas, s.BackingImageDelete))
 	backingImageActions := map[string]func(http.ResponseWriter, *http.Request) error{
-		"backingImageCleanup": s.BackingImageCleanup,
-
-		BackingImageUpload: s.fwd.Handler(s.fwd.HandleProxyRequestForBackingImageUpload, UploadParametersForBackingImage(s.m), s.BackingImageGet),
+		"backingImageCleanup":      s.BackingImageCleanup,
+		BackingImageUpload:         s.fwd.Handler(s.fwd.HandleProxyRequestForBackingImageUpload, UploadParametersForBackingImage(s.m), s.BackingImageGet),
+		"backupBackingImageCreate": s.BackupBackingImageCreate,
```

Comment:
It's about creation, suggest using `POST` for consistency with functions.

Reply:
For these three actions, they are `"POST"`, it is defined the next line.

```
	backingImageActions := map[string]func(http.ResponseWriter, *http.Request) error{
		"backingImageCleanup":      s.BackingImageCleanup,
		BackingImageUpload:         s.fwd.Handler(s.fwd.HandleProxyRequestForBackingImageUpload, UploadParametersForBackingImage(s.m), s.BackingImageGet),
		"backupBackingImageCreate": s.BackupBackingImageCreate,
	}
	for name, action := range backingImageActions {
		r.Methods("POST").Path("/v1/backingimages/{name}").Queries("action", name).Handler(f(schemas, action))
	}
```

Reply:
This has been clarified. 

`backupBackupImageCreate` is not for backing image, but rather for backing image backup. Ideally, CRUD operations suggested to be mapped to the corresponding HTTP action; however, this is only for the same domain resource.